pkgs/development/libraries/db/db-4.8.nix

```
@@ -3,8 +3,7 @@
 import ./generic.nix (args // {
   version = "4.8.30";
   sha256 = "0ampbl2f0hb1nix195kz1syrqqxpmvnvnfvphambj7xjrl3iljg0";
-  extraPatches = [ ./clang-4.8.patch ./CVE-2017-10140-4.8-cwd-db_config.patch ]
-    ++ lib.optionals stdenv.isDarwin [ ./darwin-mutexes-4.8.patch ];
+  extraPatches = [ ./clang-4.8.patch ./CVE-2017-10140-4.8-cwd-db_config.patch ./darwin-mutexes-4.8.patch ];
```

```suggestion
  extraPatches = [
    ./clang-4.8.patch
    ./CVE-2017-10140-4.8-cwd-db_config.patch
    ./darwin-mutexes-4.8.patch
  ];
```
to make diffs easier if those ever change

---

I reformatted and updated the PR.